Project/ProgrammaticUserCells/ProgrammaticUserCells/DetailViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  DetailViewController.swift
+//  ProgrammaticUserCells
+//
+//  Created by Michelle Cueva on 10/8/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+    
+    var user: User!
+    
+    lazy var userImage: UIImageView = {
```

Comment:
You can add put all these ui elements on a separate file. You can create a coco touch class file and conform to a UIView, add your elements there, initialize the class, add it to the super view, make an instance of that file on your view controller. It makes the VC some much cleaner